contrib/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.5.1-alpine
+
+ENV BEANCOUNT_INPUT_FILE ""
+ENV FAVA_OPTIONS "-H 0.0.0.0"
+ENV FINGERPRINT "3f:d3:c5:17:23:3c:cd:f5:2d:17:76:06:93:7e:ee:97:42:21:14:aa"
+ENV BUILDDEPS "libxml2-dev libxslt-dev gcc musl-dev mercurial git nodejs make g++"
+ENV RUNDEPS "libxml2 libxslt"
+
+RUN cd /root \
+        && apk add --update $BUILDDEPS $RUNDEPS \
+        && hg clone --config hostfingerprints.bitbucket.org=$FINGERPRINT https://bitbucket.org/blais/beancount \
+        && (cd beancount && hg log -l1) \
+        && python3 -mpip install ./beancount \
+        && git clone https://github.com/aumayr/fava.git \
+        && (cd fava && git log -1) \
+        && make -C fava build-js \
```

Comment:
@aumayr sorry for the delayed turn around on this. I'm having a trouble building recent versions of fava with `python3 -mpip install ./fava` - I get the above error, my python knowledge is near non-existant so it's highly possible I'm missing something obvious.

It looks like it's attempting to process the CHANGES file for some odd reason? Do pip packages have associated changelogs? Either way, it's not happy with it - any guesses as to why? Irritatingly I'm not able to reproduce this outside of the alpine container - it works fine on my box.